packages/dep_int_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018 Sonatype Inc.
```

Comment:
Technically, you own the copyright on this file, it's all you. It's a test so lol no one cares, but figured I'd point it out.

Reply:
looking over it now. It looks like all the other tests have the same copyright on the file so I'm going to leave it